GHA: PyPy3 install failing on Windows

<!--
Thank you for reporting an issue.

Follow these guidelines to ensure your issue is handled properly.

If you have a ...

1.  General question: consider asking the question on Stack Overflow
    with the python-imaging-library tag:

    * https://stackoverflow.com/questions/tagged/python-imaging-library

    Do not ask a question in both places.

    If you think you have found a bug or have an unexplained exception
    then file a bug report here.

2.  Bug report: include a self-contained, copy-pastable example that
    generates the issue if possible. Be concise with code posted.
    Guidelines on how to provide a good bug report:

    * https://stackoverflow.com/help/mcve

    Bug reports which follow these guidelines are easier to diagnose,
    and are often handled much more quickly.

3.  Feature request: do a quick search of existing issues
    to make sure this has not been asked before.

We know asking good questions takes effort, and we appreciate your time.
Thank you.
-->

### What did you do?

Rebase a PR and test on GitHub Actions.

### What did you expect to happen?

It passes, as it had pre-rebase.

### What actually happened?

It passes, except for GHA's "Test Windows / Python pypy3.6 x86", that fails at "Install PyPy".

All other PRs are also failing.

Last pass: https://github.com/python-pillow/Pillow/runs/407161674?check_suite_focus=true

First fail: https://github.com/python-pillow/Pillow/runs/408445780?check_suite_focus=true

#### The install step

```yml
    - name: Install PyPy
      if: "contains(matrix.python-version, 'pypy')"
      run: |
        curl -fsSL -o pypy3.zip "${{ matrix.pypy-url }}"
        7z x pypy3.zip "-o$env:RUNNER_WORKSPACE\"
        mv "$env:RUNNER_WORKSPACE\${{ matrix.pypy-version }}" "$env:RUNNER_WORKSPACE\${{ matrix.python-version }}"
        $env:PYTHON="$env:RUNNER_WORKSPACE\${{ matrix.python-version }}"
        # set env: pythonLocation
        Write-Host "`#`#[set-env name=pythonLocation;]$env:PYTHON" # old syntax https://github.com/actions/toolkit/issues/61
        Write-Host "::set-env name=pythonLocation::$env:PYTHON" # new syntax https://github.com/actions/toolkit/blob/5bb77ec03fea98332e41f9347c8fbb1ce1e48f4a/docs/commands.md
        New-Item -ItemType SymbolicLink -Path "$env:PYTHON\python.exe" -Target "$env:PYTHON\pypy3.exe"
        curl -fsSL -o get-pip.py https://bootstrap.pypa.io/get-pip.py
        $env:PATH = "$env:PYTHON\bin;$env:PATH"
        & $env:PYTHON\python.exe get-pip.py
      shell: pwsh
```

#### The error

```
Run curl -fsSL -o pypy3.zip "https://bitbucket.org/pypy/pypy/downloads/pypy3.6-v7.3.0-win32.zip"

7-Zip 19.00 (x64) : Copyright (c) 1999-2018 Igor Pavlov : 2019-02-21

Scanning the drive for archives:
1 file, 48009204 bytes (46 MiB)

Extracting archive: pypy3.zip
--
Path = pypy3.zip
Type = zip
Physical Size = 48009204

Everything is Ok

Folders: 214
Files: 3478
Size:       170359849
Compressed: 48009204


    Directory: D:\a\Pillow\pypy3.6

Mode                LastWriteTime         Length Name
----                -------------         ------ ----
-a---l         1/25/2020  7:56 AM              0 python.exe
WARNING: pip is being invoked by an old script wrapper. This will fail in a future version of pip.
##[error]Process completed with exit code 1.
```

### What are your OS, Python and Pillow versions?

* OS: Windows
* Python: PyPy3.6
* Pillow: master

### See also

Two weeks ago: "GHA: PyPy3 tests failing on macOS" (https://github.com/python-pillow/Pillow/issues/4350), was a problem with a newly-deployed GHA image (https://github.com/actions/virtual-environments/issues/249). Could this be the same? At least I don't see any similar changelog entries to indicate new images, like https://github.blog/changelog/2020-01-09-github-actions-updates-to-the-macos-10-15-virtual-environment/ from last time.

The problem is somewhere in our custom PyPy download and install step, so not in the GHA image.

Adding debug echos to the step:

```yml
    - name: Install PyPy
      if: "contains(matrix.python-version, 'pypy')"
      run: |
        echo 1
        curl -fsSL -o pypy3.zip "${{ matrix.pypy-url }}"
        echo 2
        7z x pypy3.zip "-o$env:RUNNER_WORKSPACE\"
        echo 3
        mv "$env:RUNNER_WORKSPACE\${{ matrix.pypy-version }}" "$env:RUNNER_WORKSPACE\${{ matrix.python-version }}"
        echo 4
        $env:PYTHON="$env:RUNNER_WORKSPACE\${{ matrix.python-version }}"
        echo 5
        # set env: pythonLocation
        echo 6
        Write-Host "`#`#[set-env name=pythonLocation;]$env:PYTHON" # old syntax https://github.com/actions/toolkit/issues/61
        echo 7
        Write-Host "::set-env name=pythonLocation::$env:PYTHON" # new syntax https://github.com/actions/toolkit/blob/5bb77ec03fea98332e41f9347c8fbb1ce1e48f4a/docs/commands.md
        echo 8
        New-Item -ItemType SymbolicLink -Path "$env:PYTHON\python.exe" -Target "$env:PYTHON\pypy3.exe"
        echo 9
        curl -fsSL -o get-pip.py https://bootstrap.pypa.io/get-pip.py
        echo 10
        $env:PATH = "$env:PYTHON\bin;$env:PATH"
        echo 11
        & $env:PYTHON\python.exe get-pip.py
        echo 12
      shell: pwsh
```

Showed the error coming from `& $env:PYTHON\python.exe get-pip.py`:

```
1
2

7-Zip 19.00 (x64) : Copyright (c) 1999-2018 Igor Pavlov : 2019-02-21

Scanning the drive for archives:
1 file, 48009204 bytes (46 MiB)

Extracting archive: pypy3.zip
--
Path = pypy3.zip
Type = zip
Physical Size = 48009204

Everything is Ok

Folders: 214
Files: 3478
Size:       170359849
Compressed: 48009204
3
4
5
6
7
8


    Directory: D:\a\Pillow\pypy3.6

Mode                LastWriteTime         Length Name
----                -------------         ------ ----
-a---l         1/25/2020  6:56 PM              0 python.exe
9
10
11
WARNING: pip is being invoked by an old script wrapper. This will fail in a future version of pip.
##[error]Process completed with exit code 1.
```

There have been some problems with the pip 20 releases this week, but anyway, https://github.com/python-pillow/Pillow/pull/4383 fixes it in another way:

I don't think PyPy was available in the GHA image when we first set this up, but it is now, and https://github.com/python-pillow/Pillow/pull/4383 fixes it by using that instead.

--- a/.github/workflows/test-windows.yml
+++ b/.github/workflows/test-windows.yml
@@ -9,7 +9,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.5", "3.6", "3.7", "3.8", "pypy3.6"]
+        python-version: ["3.5", "3.6", "3.7", "3.8", "pypy3"]
         architecture: ["x86", "x64"]
         include:
           - architecture: "x86"
@@ -18,11 +18,9 @@ jobs:
           - architecture: "x64"
             platform-vcvars: "x86_amd64"
             platform-msbuild: "x64"
-          - python-version: "pypy3.6"
-            pypy-version: "pypy3.6-v7.3.0-win32"
-            pypy-url: "https://bitbucket.org/pypy/pypy/downloads/pypy3.6-v7.3.0-win32.zip"
         exclude:
-          - python-version: "pypy3.6"
+          # PyPy does not support 64-bit on Windows
+          - python-version: "pypy3"
             architecture: "x64"
     timeout-minutes: 30
 
@@ -46,25 +44,8 @@ jobs:
           ${{ runner.os }}-${{ matrix.python-version }}-${{ matrix.architecture }}-
           ${{ runner.os }}-${{ matrix.python-version }}-
 
-    - name: Install PyPy
-      if: "contains(matrix.python-version, 'pypy')"
-      run: |
-        curl -fsSL -o pypy3.zip "${{ matrix.pypy-url }}"
-        7z x pypy3.zip "-o$env:RUNNER_WORKSPACE\"
-        mv "$env:RUNNER_WORKSPACE\${{ matrix.pypy-version }}" "$env:RUNNER_WORKSPACE\${{ matrix.python-version }}"
-        $env:PYTHON="$env:RUNNER_WORKSPACE\${{ matrix.python-version }}"
-        # set env: pythonLocation
-        Write-Host "`#`#[set-env name=pythonLocation;]$env:PYTHON" # old syntax https://github.com/actions/toolkit/issues/61
-        Write-Host "::set-env name=pythonLocation::$env:PYTHON" # new syntax https://github.com/actions/toolkit/blob/5bb77ec03fea98332e41f9347c8fbb1ce1e48f4a/docs/commands.md
-        New-Item -ItemType SymbolicLink -Path "$env:PYTHON\python.exe" -Target "$env:PYTHON\pypy3.exe"
-        curl -fsSL -o get-pip.py https://bootstrap.pypa.io/get-pip.py
-        $env:PATH = "$env:PYTHON\bin;$env:PATH"
-        & $env:PYTHON\python.exe get-pip.py
-      shell: pwsh
-
     # sets env: pythonLocation
     - name: Set up Python
-      if: "!contains(matrix.python-version, 'pypy')"
       uses: actions/setup-python@v1
       with:
         python-version: ${{ matrix.python-version }}

